BlueDBOnDisk/src/main/java/org/bluedb/disk/serialization/validation/ObjectValidation.java

```diff
@@ -144,4 +145,11 @@ private static boolean isNotEmptyCollection(Object obj) {
 		}
 		return true;
 	}
+
+	private static boolean isByteArray(Field field) {
+		if(field.getType().equals(byte[].class) || field.getType().equals(Byte[].class)) {

```

Comment:
Super small thing but you could just return the result of this conditional. If you think it looks more readable without doing that then you don't need to change it.

Reply:
fixed